packages/react/src/BottomSheet.tsx

```diff
@@ -0,0 +1,118 @@
+import { exists, OmitKeyof } from "@plainsheet/utility";
+import {
+  BottomSheetCore,
+  createBottomSheet,
+  createPlaceholderBottomSheet,
+  BottomSheetProps,
+} from "@plainsheet/core";
+export { createPlaceholderBottomSheet } from "@plainsheet/core";
+export type { BottomSheetCore } from "@plainsheet/core";
+import {
+  forwardRef,
+  ReactNode,
+  useCallback,
+  useEffect,
+  useImperativeHandle,
+  useRef,
+  useState,
+} from "react";
+import { createPortal } from "react-dom";
+
+export type PlainBottomSheetProps = {
+  children: ReactNode;
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  mountingPoint?: Element | null;
+} & CoreProps;
+type CoreProps = OmitKeyof<BottomSheetProps, "content", "safely">;
```

Comment:
Remove safely please

Reply:
Caught some type errors thanks to this. Thank you.